after/ftplugin/xml.vim

```diff
@@ -1,12 +1,6 @@
 if gradle#isGradleProject()
-
-  call gradle#setClassPath()
```

Comment:
I'll be sure to retain the existing classpath system then and just default to using JavaComplete2's when it exists